<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HxInformaion</class>
 <widget class="QWidget" name="HxInformaion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1670</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="power_left">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/engine0.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="power_right">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>80</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/engine0.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>1#推进主机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>20</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>2#推进主机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar_left">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>71</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {   
border: 2px solid rgb(0, 255, 255);   
border-radius: 5px;   
background-color: #06061d;
}
QProgressBar::chunk {   
background-color: rgb(0, 255, 255);
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar_right">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>240</y>
     <width>71</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {   
border: 2px solid rgb(0, 255, 255);   
border-radius: 5px;   
background-color: #06061d;
}
QProgressBar::chunk {   
background-color: rgb(0, 255, 255);
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="power_left_gk_lab">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>0.99</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>1#主机工况</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="power_right_gk_lab">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>250</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>0.99</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>2#主机工况</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar_ym">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>230</y>
     <width>71</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {   
border: 2px solid rgb(0, 255, 255);   
border-radius: 5px;   
background-color: #06061d;
}
QProgressBar::chunk {   
background-color: rgb(0, 255, 255);
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>700</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>主油门指令</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ym_lab">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>700</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>700</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>剩余燃油</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar_ry">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>230</y>
     <width>71</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {   
border: 2px solid rgb(0, 255, 255);   
border-radius: 5px;   
background-color: #06061d;
}
QProgressBar::chunk {   
background-color: rgb(0, 255, 255);
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ry_lab">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>700</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>1#主机档位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="power_left_dw">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>1档</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="power_right_dw">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>250</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>1档</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>2#主机档位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>800</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/leftOne.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>740</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>1#发电机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele_2">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>800</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/rightOne.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>740</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>2#发电机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>740</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>1#电力发电机连接状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele_state_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>740</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>740</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>2#电力发电机连接状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele_state_2">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>740</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>850</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>电压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele1_V">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>850</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>300V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele1_A">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>850</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>200A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>850</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>电流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele2_A">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>840</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>200A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>840</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>电流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele2_V">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>840</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>300V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>840</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>电压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>700</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>副油门指令</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ym_lab_1">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>700</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar_ym_1">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>230</y>
     <width>71</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {   
border: 2px solid rgb(0, 255, 255);   
border-radius: 5px;   
background-color: #06061d;
}
QProgressBar::chunk {   
background-color: rgb(0, 255, 255);
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ele_an_state">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>810</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(0, 150, 255);
color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>岸电</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele_an_A">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>910</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>200A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>880</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>岸电电流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ele_an_V">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>910</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>300V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>880</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>岸电电压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MeterWidget" name="duo_widget" native="true">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="MeterWidget1" name="duo1_widget" native="true">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>主舵指令</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>180</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 150, 255);</string>
   </property>
   <property name="text">
    <string>副舵指令</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>ry_lab</zorder>
  <zorder>ym_lab_1</zorder>
  <zorder>ele1_V</zorder>
  <zorder>ele_state_1</zorder>
  <zorder>ele_state_2</zorder>
  <zorder>power_left_dw</zorder>
  <zorder>power_left</zorder>
  <zorder>power_right</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>pBar_left</zorder>
  <zorder>pBar_right</zorder>
  <zorder>power_left_gk_lab</zorder>
  <zorder>label_4</zorder>
  <zorder>power_right_gk_lab</zorder>
  <zorder>label_5</zorder>
  <zorder>pBar_ym</zorder>
  <zorder>ym_lab</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>pBar_ry</zorder>
  <zorder>label_8</zorder>
  <zorder>power_right_dw</zorder>
  <zorder>label_9</zorder>
  <zorder>ele_1</zorder>
  <zorder>label_3</zorder>
  <zorder>ele_2</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>ele1_A</zorder>
  <zorder>label_14</zorder>
  <zorder>ele2_A</zorder>
  <zorder>label_15</zorder>
  <zorder>ele2_V</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>pBar_ym_1</zorder>
  <zorder>ele_an_state</zorder>
  <zorder>ele_an_A</zorder>
  <zorder>label_19</zorder>
  <zorder>ele_an_V</zorder>
  <zorder>label_20</zorder>
  <zorder>duo_widget</zorder>
  <zorder>duo1_widget</zorder>
  <zorder>label_18</zorder>
  <zorder>label_21</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MeterWidget</class>
   <extends>QWidget</extends>
   <header>meterwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MeterWidget1</class>
   <extends>QWidget</extends>
   <header>meterwidget1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
